src/components/Auth/ChangePassword/ChangePassword.react.js

```diff
@@ -281,48 +287,54 @@ export default class ChangePassword extends Component {
                                 onChange={this.handleChange}
                                 inputStyle={inputStyle}
                                 errorText={this.passwordErrorMessage}
-                                underlineFocusStyle={underlineFocusStyle}
-                                floatingLabelStyle={floatingLabelStyle}
-                                visibilityIconStyle={{color:themeForegroundColor}}
-                                floatingLabelFocusStyle={underlineFocusStyle}
-                                floatingLabelText={<Translate text="Current Password" />} />
+                                underlineStyle={{display: 'none'}}
+                                disableButton={true}
+                                visibilityButtonStyle={{'display':'none'}}
+                                visibilityIconStyle={{'display':'none'}} />
+                            <div className='forgot'>
+                            <a href='https://accounts.susi.ai/forgotpwd'>
+                              Forgot your password?
+                            </a>
+                        </div>
                         </div>
+                        <br />
+                        <div style={labelStyle}>New Password</div>
                         <div className={PasswordClass.join(' ')}>
                             <PasswordField
                                 name="newPassword"
                                 style={fieldStyle}
                                 value={this.state.newPasswordValue}
                                 onChange={this.handleChange}
                                 inputStyle={inputStyle}
-                                floatingLabelStyle={floatingLabelStyle}
-                                visibilityIconStyle={{color:themeForegroundColor}}
                                 errorText={this.newPasswordErrorMessage}
-                                underlineFocusStyle={underlineFocusStyle}
-                                floatingLabelFocusStyle={underlineFocusStyle}
-                                floatingLabelText={<Translate text="New Password" />} />
-                              <div className="ReactPasswordStrength-strength-bar" />
-                              <div>
+                                underlineStyle={{display: 'none'}}
+                                disableButton={true}
+                                visibilityButtonStyle={{'display':'none'}}
+                                visibilityIconStyle={{'display':'none'}} />
+                              <div className="ReactPasswordStrength" />
+                              <div >
                                   {this.state.newPasswordStrength}
                               </div>
                         </div>
+                        <br />
```

Comment:
I think we can find a better solution than using this. :sweat_smile:

Reply:
@anshumanv I think a single br tag should not be a problem. The UI works smoothly with it. Removing this will only increase the complexity. Thanks.